modules/protocol/unreal4.cpp

```diff
@@ -13,14 +13,26 @@
 #include "modules/cs_mode.h"
 #include "modules/sasl.h"
 
+typedef std::map<Anope::string, Anope::string> ModData;
 static Anope::string UplinkSID;
 
+class pipesepstream : public sepstream
+{
+ public:
+	/** Initialize with separator
+	 */
+	pipesepstream(const Anope::string &source, bool allowempty = false) : sepstream(source, '|', allowempty) { }
+};
+
 class UnrealIRCdProto : public IRCDProto
 {
  public:
-	UnrealIRCdProto(Module *creator) : IRCDProto(creator, "UnrealIRCd 4+")
+	PrimitiveExtensibleItem<ModData> ClientModData;
+	PrimitiveExtensibleItem<ModData> ChannelModData;
+
+	UnrealIRCdProto(Module *creator) : IRCDProto(creator, "UnrealIRCd 4+"), ClientModData(creator, "ClientModData"), ChannelModData(creator, "ChannelModData")
 	{
-		DefaultPseudoclientModes = "+Soiq";
+		DefaultPseudoclientModes = "+SoiqB";
```

Comment:
Most protocol modules have these in alphabetical order.

```suggestion
		DefaultPseudoclientModes = "+BioqS";
```